src/test/java/racingcar/controller/WebRacingGameControllerTest.java

```diff
@@ -138,6 +128,47 @@ void shouldReturnWinnersAndRacingCarsWhenRequestCorrectly() {
         }
     }
 
+    @Nested
+    @DisplayName("/plays :GET")
+    class PlaysGet {
+
+        @Test
+        @DisplayName("요청을 보낼 수 있다")
+        void shouldBeAbleWhenRequest() {
+            RestAssured.given().log().all()
+                    .when().get("/plays")
+                    .then().log().all()
+                    .statusCode(HttpStatus.OK.value());
+        }
+
+        @Test
+        @DisplayName("요청 시, 모든 우승자와 자동차 정보를 규격에 맞게 반환한다")
+        void shouldReturnWinnersAndRacingCarsWhenRequest() {
+            // 자동차 경주 게임을 두 번 진행한다
+            Model model = new ConcurrentModel();
+            model.addAttribute("names", "브리,토미,브라운");
+            model.addAttribute("count", "3");
```

Comment:
```suggestion
            RacingGameRequest request = new RacingGameRequest("브리,토미,브라운", 3);
```
controller에서 modelAttribute가 아닌 requestBody를 통해서 받고 있는데
테스트도 request 객체를 만들어 테스트하는게 더 좋지 않을까여?

Reply:
감사합니다!

말씀해주신대로 변경하려고 해보았는데요,
다만 RacingGameRequest를 사용하는 경우 `count`필드가 보내지지 않거나 
`names` 필드가 보내지지 않는 경우에 대한 테스트가 안되는 것 같습니다.

`RacingGameRequest`의 경우 객체이고 필드가 존재하지 않다는 것을 지정할 수 없으니까요.

이런 부분에서 Controller request에 대한 테스트는 Model로 순수하게
JSON 형식으로 보내는 게 좋다는 생각인데 영이는 이에 대해 어떻게 생각하시는지 궁금합니다!

Reply:
그렇네요! 
modelAttribute를 쓰는 경우는 거의 없어서
이건 조금만 더 고민해볼게요!

Reply:
저도 한번 고민해보겠습니다 😀